packages/calcite-components/src/components/panel/panel.scss

```diff
@@ -2,11 +2,30 @@
- * @prop --calcite-panel-header-border-block-end: Specifies the component header's block end border.
+
+ * @prop --calcite-panel-corner-radius: Specifies the corner radius of the component.
+ * @prop --calcite-panel-heading-text-color: Specifies the text color of the component's `heading`.
+ * @prop --calcite-panel-description-text-color: Specifies the text color of the component's `description`.
+ * @prop --calcite-panel-border-color: Specifies the border color of the component.
+
  * @prop --calcite-panel-background-color: Specifies the background color of the component.
+ * @prop --calcite-panel-header-background-color: Specifies the background color of the component's footer.
+ * @prop --calcite-panel-footer-background-color: Specifies the background color of the component's footer.
+
+ * @prop --calcite-panel-space: Specifies the padding of the component's `"unnamed (default)"` slot.
+ * @prop --calcite-panel-header-content-space: Specifies the padding of the `"header-content"` slot.
+ * @prop --calcite-panel-footer-space: Specifies the padding of the component's footer.
+
+ * @prop --calcite-panel-content-space: [Deprecated] Specifies the padding of the component's content. Use `--calcite-panel-space` instead.
+ * @prop --calcite-panel-footer-padding: [Deprecated] Specifies the padding of the component's footer. Use `--calcite-panel-footer-space` instead.
+ * @prop --calcite-panel-header-border-block-end: [Deprecated] Specifies the component header's block end border. Use `--calcite-panel-border-color` instead.
+
+ * @prop --calcite-action-background-color: Specifies the background color of the component's `closable`, `collapsible`, and `back` Action components. Will apply to any slotted Calcite Action components.
+ * @prop --calcite-action-background-color-hover: Specifies the background color of the component's `closable`, `collapsible`, and `back` Action components when hovered. Will apply to any slotted Calcite Action components.
+ * @prop --calcite-action-background-color-pressed: Specifies the background color of the component's `closable`, `collapsible`, and `back` Action components when pressed. Will apply to any slotted Calcite Action components.
+ * @prop --calcite-action-text-color-hover: Specifies the text and icon color of the component's `closable`, `collapsible`, and `back` Action components when hovered. Will apply to any slotted Calcite Action components.
+ * @prop --calcite-action-text-color-pressed: Specifies the text and icon color of the component's `closable`, `collapsible`, and `back` Action components when pressed. Will apply to any slotted Calcite Action components.
+ * @prop --calcite-popover-border-color: Specifies the border color of the component's internally rendered Popover component, which is rendered within an Action Menu when slotted Actions are present in the `header-actions-end` slot. Will apply to any slotted Calcite Popover components.
  */
 
 :host {
```

Comment:
`border-radius` needs to be added to the :host or the component doesn't look right.

Reply:
Yeah I don't know if this component needs `@extend %component-host;` at all which places that visible background that isn't clipped. Might be able to remove that and not have to set the radius on :host

Reply:
I was able to just take what was needed from `%component-host` and not set a background color on :host. Could you verify: https://www.chromatic.com/build?appId=6266d45509d7eb004aa254fb&number=6466

Reply:
Going to merge in.